<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RegisterWindow</class>
 <widget class="QMainWindow" name="RegisterWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="label">
      <property name="font">
       <font>
        <pointsize>32</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Food Rating</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLineEdit" name="firstNameLineEdit"/>
      </item>
      <item>
       <widget class="QLineEdit" name="middleNameLineEdit"/>
      </item>
      <item>
       <widget class="QLineEdit" name="lastNameLineEdit"/>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QLineEdit" name="emailLineEdit"/>
    </item>
    <item>
     <widget class="QLineEdit" name="passwordLineEdit"/>
    </item>
    <item>
     <widget class="QLineEdit" name="passwordConfirmLineEdit"/>
    </item>
    <item>
     <widget class="QPushButton" name="registerButton">
      <property name="text">
       <string>Register</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
